Overview.src.html

```diff
@@ -1589,6 +1589,17 @@ <h3 id="should-response-to-request-be-blocked-due-to-mime-type?"><dfn title="sho
  <li><p>Return <var>reader</var>.
 </ol>
 
+<p>To <dfn title=concept-read-chunk-from-ReadableStream>read a chunk</dfn> from a
+<code title=concept-ReadableStream>ReadableStream</code> object with <var>reader</var>, run these
+steps:
+
+<ol>
+ <li><p>Let <var>promise</var> be the result of calling
+ <code data-anolis-spec=streams>ReadFromReadableStreamReader</code>(<var>reader</var>).
+
+ <li><p>Return <var>promise</var>.
+</ol>
```

Comment:
I think we should inline this and the other operation you introduce by replacing "run these steps" with "return the result of calling ..."